src/main/g8/src/main/proto/$package$/counter_domain.proto

```diff
@@ -1,15 +0,0 @@
-syntax = "proto3";
-
-package $package$.domain;
```

Comment:
looks like this file was deleted, shouldn't it be moved?

Reply:
Sure, my bad. Fixed now.